cmd/summary.go

```diff
@@ -101,7 +105,7 @@ func summary(cmd *cobra.Command, args []string) {
 	var modelResults jsonResults
 
 	for w := 1; w <= workers; w++ {
-		go func(w int, modIndex <-chan int, results chan<- modelResult) {
+		go func(_ int, modIndex <-chan int, results chan<- modelResult) {
```

Comment:
could probably just drop this instead - since w isn't used anywhere and just simplify the func to modIndex,results.

Reply:
Yeah, a lot of them can be dropped, so long as we didn't unintentionally not use them. Hence the TODO